Mouse.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "Coordinate.h"
+#include "Direction.h"
+#include "Hardware.h"
+#include "Maze.h"
+#include "Path.h"
+
+class Mouse {
+  public:
+    Mouse();
+
+    void mapMaze();
+    void runMaze();
+
+    // In millimeter per second.
+    void setMappingSpeed(unsigned mmps);
+    void setRunningSpeed(unsigned mmps);
+
+    void reset();
+
+  private:
+    void followPath(const Path &path);
```

Comment:
should we add a function void goToCell(CellCoordinate cell); that creates a path from the current cell to the cell using the path finder and calls followPath?